src/caf/toolkit/cost_utils.py

```diff
@@ -252,6 +303,7 @@ def from_file(
         max_col: str = "max",
         avg_col: str = "ave",
         trips_col: str = "trips",
+        weighted_avg_col: str = "weighted_ave",
```

Comment:
Missing docstring for this. Do we also want to make this optional and default to the same as the average bin?